### ISSUE TITLE
Keep Data Intact will not save tag information of a block

### ISSUE THREAD

**Issue by ErikSteiner** [Khroki/MCEdit-Unified#589]
Hi guys, I am currently using MCEdit 1.5.1 and I try to update some blocks of the Mod Carpenters Blocks. I have the problem that I want to change the block ID of those blocks via the Editor.

As you can see here: http://i.imgur.com/BlpRMPo.png Those blocks contain a lot of information that will be then used to display them correctly ingame. They contain information about the rotation and the material that is used etc..

When I try to use the Replace Tool of MCEdit and check the box "Keep Data Intact" these TAG information will vanish after I changed their block ID.

Is this a bug? I am pretty sure that it was working one time..

**Comment by Rubisk** [comment]
Keep Data intact is supposed to only change the data/meta value. Any NBT gets deleted, as most nbt is ID-specific.

**Comment by ErikSteiner** [comment]
Ah ok.. I updated my Server and I exported the map into a new one. The problem was, that several Block IDs changed. This was not very critical because I changed them via MCEdit, but trying this with the Carpenters' Blocks was a nightmare. Thank god I found a way to do this anyway. Jigarbov's Block Filter is what I need. It will change the Block IDs but the Entities will be left untouched.

This issue seems to be solved then.
